01_fib/sriyapaka/main_test.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestFib(t *testing.T) {
+
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	expected := strconv.Quote("1\n1\n2\n3\n5\n8\n13\n")
+	actual := strconv.Quote(buf.String())
+	if expected != actual {
+		t.Errorf("unexpected result - fib function")
```

Comment:
It would probably help if you printed the expected and actual outputs here, in the event this error is triggered you won't actually know why. The same goes for when this was used again in `TestNegativeSeries()`.

Further to that it would probably help if you looked at using table driven tests, this will enable you to merge the `TestNegativeSeries` test case into your `TestFib` test case. From the slides, this should give you a good reference on how to approach this: https://dave.cheney.net/2019/05/07/prefer-table-driven-tests